Microsoft.Toolkit.Uwp.UI.Controls/GridSplitter/GridSplitter.cs

```diff
@@ -0,0 +1,170 @@
+﻿using Windows.UI.Core;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    /// Represents the control that redistributes space between columns or rows of a Grid control.
+    /// </summary>
+    [TemplatePart(Name = SPLITTERRNAME, Type = typeof(Thumb))]
+    [TemplatePart(Name = ICONDISPLAYNAME, Type = typeof(Thumb))]
+    public partial class GridSplitter : Control
+    {
+        private const string SPLITTERRNAME = "Splitter";
+        private const string ICONDISPLAYNAME = "IconDisplay";
+
+        // Symbol GripperBarVertical in Segoe MDL2 Assets
```

Comment:
I think it should be content, with a visual state flipping between vertical and horizontal content. We should make these symbols the default ones though.